deepmd/pd/model/atomic_model/base_atomic_model.py

```
@@ -0,0 +1,580 @@
+            fparam=fparam,
+            aparam=aparam,
+            comm_dict=comm_dict,
+        )
+        ret_dict = self.apply_out_stat(ret_dict, atype)
+
+        # nf x nloc
+        atom_mask = ext_atom_mask[:, :nloc].astype(paddle.int32)
+        if self.atom_excl is not None:
+            atom_mask *= self.atom_excl(atype)
+
+        for kk in ret_dict.keys():
```

Ruff

Use `key in dict` instead of `key in dict.keys()`

Remove `.keys()`

(SIM118)